jakarta-eclipse/src/org/jakarta/codeAction/AbtractJavaContext.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * This is a reuse from 
+ * https://github.com/eclipse/lsp4mp/blob/master/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/AbtractJavaContext.java
+ * 
+*******************************************************************************/
```

Comment:
We should leave the header from the original file we retrieved this from, instead add a javadoc comment on the class crediting where we got this from